cpp/src_prims/sparse/coo.h

```diff
@@ -436,7 +448,7 @@ void coo_row_count(COO<T> *const in, int *results, cudaStream_t stream = 0) {
 }
 
 template <int TPB_X, typename T>
-__global__ void coo_row_count_nz_kernel(int *const rows, T *const vals, int nnz,
+__global__ void coo_row_count_nz_kernel(const int *rows, const T *vals, int nnz,
```

Comment:
Another custom kernel.

Reply:
This is a degree computation on sparse data formats. It’s a primitive.